driver/util.d

```
@@ -102,6 +103,39 @@ wstring buildDebugMessage(TList...)(auto ref TList vs) {
     return wtext(joiner(rngOfVs, " "));
 }
 
+string getTextInput(string fileTemplate = "") {
+    import std.process : execute;
+
+    auto tempFile = makeTempFile(fileTemplate);
+    scope (exit) { tempFile.remove; }
+    if (auto rc = execute(["notepad.exe", tempFile]).status != 0) {
+        throw new OdbcException(StatusCode.GENERAL_ERROR, "Notepad returned bad return code "w ~ wtext(rc));
+    }
+    return tempFile.readText!string();
+}
+
+string makeTempFile(string content = "")
+out(tempFile) {
+    dllEnforce(tempFile.exists, "File must exist");
+} body {
+    import std.random : uniform;
+
+    Exception eSaved;
+    for (auto i = 0; i < 10; ++i) {
+        try {
+            //As long as this driver is single-threaded I don't believe we're likely
+            //to find this problematic.
+            auto tempFile = tempPath ~ "temp" ~ text(uniform(0, 1_000_000));
+            if (tempFile.exists) {
+                continue;
+            }
+            tempFile.write(content);
+            return tempFile;
+        } catch (Exception e) { eSaved = e; }
+    }
+    assert(false, "Failed to make temp file" ~ to!string(eSaved));
```

I believe this should throw an exception